tools/Session.php

```diff
@@ -31,6 +33,11 @@ public function setKey($key, $value){
 
 	}
 
+	public function setValueKey($key, $valueKey, $value){
+
+		$_SESSION[$key][$valueKey] = $value;
```

Comment:
Codacy found an issue: [Direct use of $_SESSION Superglobal detected.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)